BUILD.md

```diff
@@ -21,7 +21,7 @@
 These dependencies need not be installed manually. They are automatically downloaded on the platforms where they are required.
 - [Bullet Physics Engine](https://github.com/bulletphysics/bullet3/releases):  2.83  
 - [glm](https://glm.g-truc.net/0.9.8/index.html):  0.9.8  
-- [Oculus SDK](https://developer.oculus.com/downloads/):   1.11 (Windows) / 0.5 (Mac)  
+- [Oculus SDK](https://developer.oculus.com/downloads/):   23.0 (Android)  
```

Comment:
Oculus SDK is used on Windows, unless it got removed in the last couple of months which is doubt.

Reply:
Yes, I think this PR shouldn't be changing anything to do w.r.t. Oculus?

Reply:
Good call out, I wasn't aware of any specific Oculus support in Windows and Mac, and thought this from the pre-Oculus (Rift) days.  As this is the general build doc, it could also apply to Android so I put in the Oculus SDK we updated on Android.  I can revert this change if Windows/Mac support was for something like Oculus controllers only.